assets/script/gamelogic.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable   import/extensions */
+function gameLogic() {
+  return {
+    winArray: [[1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9],
+      [1, 4, 7],
+      [2, 5, 8],
+      [3, 6, 9],
+      [1, 5, 9],
+      [3, 5, 7]],
+
+    cellFull: false,
+
+    gameBoard: ['', '', '', '', '', '', '', '', ''],
+
+    ValiditePosition(e) {
+      const errors = document.querySelector('.errors');
+      let res = true;
+
+      if (e.target.classList.contains('clicked')) {
+        errors.classList.remove('d-none');
+        errors.innerText = 'That position is taken!';
+        res = false;
```

Comment:
[OPTIONAL] - **this is marked as optional because it wasn't mentioned in the previous reviews but it's a requirement**, so I highly recommend you to do this
- Keep the application logic separated from DOM manipulation tasks see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/dom_logic_separation_requirement.png) from Pathwright
***There are so many DOM manipulation tasks in all your js files, please separate each of them*